p2p_wallet/Coordinators/TabBar/TabBarCoordinator.swift

```diff
@@ -248,8 +250,14 @@ final class TabBarCoordinator: Coordinator<Void> {
         case .crypto:
             self.handleAction(.receive)
         case .transfer:
-            // TODO: Put Bank transfer here
-            break
+            let service: BankTransferService = Resolver.resolve()
```

Comment:
I thought that top up coordinator should send this data to BankTransferCoordinator?

Reply:
TopUpCoordinator is responsible only for opening Actions screen